crates/extensions/c8y_http_proxy/src/handle.rs

```diff
@@ -27,16 +31,30 @@ impl C8YHttpProxy {
         C8YHttpProxy { c8y }
     }
 
-    /* Will be used by the mapper
+    pub async fn get_jwt_token(&mut self) -> Result<String, C8YRestError> {
+        let request: C8YRestRequest = GetJwtToken.into();
+        match self.c8y.await_response(request).await? {
+            Ok(C8YRestResponse::EventId(id)) => Ok(id),
+            unexpected => Err(unexpected.into()),
+        }
+    }
+
+    pub async fn url_is_in_my_tenant_domain(&mut self, url: &str) -> Result<bool, C8YRestError> {
+        let request: C8YRestRequest = IsUrlInCurrentDomain { url: url.into() }.into();
+        match self.c8y.await_response(request).await? {
+            Ok(C8YRestResponse::Bool(is)) => Ok(is),
+            unexpected => Err(unexpected.into()),
+        }
+    }
```

Comment:
This can be removed as the mapper is using it's own c8y-end-point to make this check.